### PR TITLE
Started the documentation file and added login page docs

### PR DESCRIPTION
**Pull request by vynleran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,78 @@
...
+**SignupTabFragment.java**
+- Class SignupTabFragment
+  - Method
+    - onCreate: handles the UI animcations of the signup tab
+    - signupUser: handles validating the email, password, username, and confirm password of the user and then signing them up using Firebase authentication.
+
+**User.java**
+- Class User
+  - Method
+    - User: the constructor of the class which handles creating the user object for writing to Firebase. Note: DO NOT SET THE CLASS VARIABLES TO PRIVATE. This will break firebase since it needs public variables for serialization.
```
Comment (LewisJordann) [inline README.md:67]: I dislike this un-tabbed line

Comment (LewisJordann) [reply README.md:67]: :)

**Review by LewisJordann — APPROVED** [review]
Documentation looks good!

File: README.md (hunk 1 above)
Comment (vynleran) [reply README.md:67]: Not sure why github is messing it up, but VScode was fine with it. I'll push and if there're any issues I'll fix it

**Pull request merged**